src/lib/fs.js

```diff
@@ -23,4 +25,18 @@ const fileExistsAsync = async filePath => {
   }
 }
 
-module.exports = { statAsync, readFileAsync, readFileAsyncCatchError, writeFileAsync, fileExistsAsync }
+const mkdirRecursiveSync = path => makeDir.sync(path)
+const mkdirRecursiveAsync = async path => makeDir(path)
```

Comment:
```suggestion
const mkdirRecursiveAsync = path => makeDir(path)
```

Reply:
weird, eslint is not giving me a warning about this locally.

Reply:
I just added that rule in https://github.com/netlify/cli/pull/1278 as a follow up to this PR

Reply:
I thought eslint was being crazy or something, it makes sense now. I will rebase this on top of master and fix the lint erros.